projects/containerd/containerd/ATTRIBUTION.txt

```diff
@@ -970,39 +967,6 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 ------
 
-** github.com/miekg/pkcs11; version v1.1.1 --
```

Comment:
this happens every time. there is going to be something different about the host in codebuild in prow that is pulling this in or not. ill figure it out at some point, for now its fine for it to flip back and forth.